public/src/inverted-index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * InvertedIndex class with constructor
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.indexMap = {};
+  }
+  /**
+   * Create index
+   * @function
+   * @param {Array} jsonArray [json array]
+   * @return
+   */
+  createIndex(jsonArray) {
```

Comment:
You should probably change the variable name, avoid using data types in variable names. Check  naming section in [https://github.com/andela/code-review-guidelines/tree/master/coding-style-guides](https://github.com/andela/code-review-guidelines/tree/master/coding-style-guides)